### ISSUE TITLE
'Unity' does not show up as a Build Step

### ISSUE THREAD

**Issue by Soft-Shock** [mindcandy/Teamcity-unity3d-build-runner-plugin#16]
'Unity' does not show up as a Build Step. I didn't have any errors in the build, and the plugin shows up in the TeamCity configured plugins list.

This is related to issue 15:
https://github.com/mindcandy/Teamcity-unity3d-build-runner-plugin/issues/15

Built with ant 1.9.4 on Windows 8.1 64-bit

**Comment by egorulshin** [comment]
Try to check the teamcity log on the server. Maybe some errors occur during plugin starting.

I had the same issue. It was related to version of java on the server. I built the plugin with using JDK1.8, but there is JDK 1.7 on the server. I've got UnsupportedClassVersionError exception which were logged in teamcity-server.log. I rebuilt the plugin with JDK1.6 and reinstalled it. After this it started work without any errors and the plugin appeared in the build steps.
